server/app/routes/api/v1/locations/index.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+var router = require('express').Router();
+var _ = require('lodash');
+
+const sequelizeHandlers = require( 'sequelize-handlers' );
+const bluebird = require( 'bluebird' );
+const utils = require('../../../utils');
+const {ensureAuthenticated, ensureIsAdmin, Credentials, respondWith404, _err, db} = utils;
+const Location = db().models.location;
+const Resource = Location;
```

Comment:
That was another version of doubling --> why rename location to resource. It can be named resource in the actual function, but they don't have to all be the same thing going in